src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1420,17 +1420,26 @@ internal void LoadNodeLibrary(Assembly assem, bool suppressZeroTouchLibraryLoad
            // I think this is consistent with the current behavior - IE today - if a nodeModel exists in an assembly, the rest of the assembly 
            // is not imported as ZT - the same will be true if the assembly contains a NodeViewCustomization.
 
-           // TODO(mjk) draw up matrix of current behaviors which nodeLib flag can control.
-            if (!NodeModelAssemblyLoader.ContainsNodeModelSubType(assem)
-                && !(NodeModelAssemblyLoader.ContainsNodeViewCustomizationType(assem)))
+            bool nodeModelOrNodeView;
+            try
+            {
+                nodeModelOrNodeView = !NodeModelAssemblyLoader.ContainsNodeModelSubType(assem)
+                && !(NodeModelAssemblyLoader.ContainsNodeViewCustomizationType(assem));
+            } 
+            catch (Exception ex) {
+                throw new Exceptions.LibraryLoadFailedException(assem.Location, ex.Message);
+            }
+
+            // TODO(mjk) draw up matrix of current behaviors which nodeLib flag can control.
+            if (nodeModelOrNodeView)
```

Comment:
there are 2 ways to go about this problem.
This is the one with the least code changes (safest for a hotfix I think).
In this solution we simply throw a specific type of Exception: `LibraryLoadFailedException`
This type of exception is already handled by PackageLoader and is interpreted as an acceptable failure whcih allows the pkg load operation to continue.

Another solution would be to write more code in the PackageLoader to handle exceptions in a smarter way...more changes would be needed

Reply:
so this solution still relies on enumeration order of the binaries, or was that not actually an issue?

Reply:
Yes..enumeration is still an issue. That would need to be handled on the Loader side.
I'll add that code to the PR..(I forgot about that part)